### ISSUE TITLE
Fix constructors' doxygen as they can't `@inheritdoc`

### ISSUE THREAD

**Issue by boobaa** [Pronovix/password_enhancements#22]
Affected classes:
- [ ] `\Drupal\password_enhancements\Entity\Storage\PolicyEntityStorage`
- [ ] `\Drupal\password_enhancements\Entity\Storage\ConstraintEntityStorage`
- [ ] `\Drupal\password_enhancements\Entity\Policy`
- [ ] `\Drupal\password_enhancements\Entity\PolicyListBuilder`
- [ ] `\Drupal\password_enhancements\Entity\Constraint`
- [ ] `\Drupal\password_enhancements\Entity\ConstraintListBuilder`

**Comment by mxr576** [comment]
I started to ignore these because everybody ignored this comment on PR reviews, but we should have a policy that `@inheritdic` should not be used in constructors. Yes, I know, documenting the same things several times and keeping the docblock up to date is boring, but it worth the effort.

**Comment by tnagyce** [comment]
Is it enough if you just mention the extra parameters? I mean like

```
{@inherhitdoc} 

@param string param5
@param string param6
...
```

**Comment by mxr576** [comment]
Nope, you should describe all again ;S

**Comment by tnagyce** [comment]
Not all, I meant only the additional parameters.

**Comment by mxr576** [comment]
and I have just said you should describe _all_ constructor parameters. Even if those that you inherited from parent.

**Comment by tnagyce** [comment]
Right, okay.

**Comment by boobaa** [comment]
Closed by #57.
